### ISSUE TITLE
vs-select won't open on mouse click event 

### ISSUE THREAD

**Issue by OmarBish** [lusaxweb/vuesax#883]
Vs-select won't open on when I click on it using the mouse (to be more precise it open and close instantly an don't give the user any ability to choose any item)  but it will open perfectly if I navigate to the field using the keyboard. 
this problem was reported to me throw 3 different users all facing the same problem. 

Extra info:
- all of them used chrome but I don't know which version yet. 
- vuesax version: ^3.11.17
- Node.js version : v12.16.3 
- package mager : npm 6.14.4

**Comment by otaviocardosoneto** [comment]
Hello,
Any resolution?

**Comment by OmarBish** [comment]
No, I wasn't able to solve it which is why I moved away to another select component as a solution until this problem is fixed.

**Comment by mauroferro9** [comment]
Hello,
Probably is too late, but today I was looking for the same issue and I found this, which worked for me: https://github.com/lusaxweb/vuesax/issues/770#issuecomment-649821699
